tests/mock_callable_testslide.py

```diff
@@ -179,6 +179,18 @@ def patching_private_functions_with_allow_private(self):
         ).and_assert_called_once()
         t._privatefun()
 
+    @context.example
+    def patching_typed_functions_raises_typeerror_with_invalid_responsetype(self):
+        t = TargetStr()
+        with self.assertRaises(TypeError):
```

Comment:
How is this error message looking like? I'm wondering if it needs any tweaking, or if it is good enough as it comes from typeguard.